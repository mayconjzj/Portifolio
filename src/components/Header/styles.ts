import tw from 'tailwind-styled-components';

export const Header = tw.header`
  flex
  justify-between
  items-center
  h-[100px]
  px-[7%]
`;
