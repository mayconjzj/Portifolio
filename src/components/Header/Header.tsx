import { LogoConfig } from '@/config/Logo';
import { MainNavConfig } from '@/config/MainNav';

import { Logo } from '@/components/Logo';
import { MainNav } from '@/components/MainNav';

import * as S from './styles';

export const Header = () => {
  return (
    <S.Header>
      <Logo ContentLogo={LogoConfig.ContentLogo} />

      <MainNav items={MainNavConfig} />
    </S.Header>
  );
};
