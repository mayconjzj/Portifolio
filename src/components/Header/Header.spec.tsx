import { render } from '@testing-library/react';

import { Header } from './Header';

describe('Header', () => {
  it('should render the header', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toBeTruthy();

    expect(asFragment()).toMatchSnapshot();
  });
});
