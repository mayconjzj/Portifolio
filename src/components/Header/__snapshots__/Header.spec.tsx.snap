// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the header 1`] = `
<DocumentFragment>
  <header
    class="flex justify-between items-center h-[100px] px-[7%]"
  >
    <div
      class="flex justify-center flex-col"
    >
      <span
        class="font-black text-[23px] text-blue-500"
      >
        MAYCON JZJ
      </span>
      <span
        class="text-blue-900"
      >
        Desenvolvedor Front-End
      </span>
    </div>
    <nav
      class=""
    >
      <ul
        class="flex gap-x-7"
      >
        <li>
          <a
            class="text-[#222] hover:text-blue-500 transition delay-150 cursor-pointer font-medium"
          >
            Home
          </a>
        </li>
        <li>
          <a
            class="text-[#222] hover:text-blue-500 transition delay-150 cursor-pointer font-medium"
          >
            Sobre mim
          </a>
        </li>
        <li>
          <a
            class="text-[#222] hover:text-blue-500 transition delay-150 cursor-pointer font-medium"
          >
            Projetos
          </a>
        </li>
        <li>
          <a
            class="text-[#222] hover:text-blue-500 transition delay-150 cursor-pointer font-medium"
          >
            Contato
          </a>
        </li>
      </ul>
    </nav>
  </header>
</DocumentFragment>
`;
